test-fixtures/ci/convert_junit_to_markdown.py

```diff
@@ -0,0 +1,188 @@
+import getopt, sys
+import xml.etree.ElementTree as ET
+import json
+from blockkit import Context, Divider, Header, Message, Section
+
+# Modified from junit_to_markdown
+# https://github.com/stevengoossensB/junit_to_markdown/tree/main
+
+def parse_junit_xml(file_path):
+    """
+    Parses a JUnit XML file and extracts test suite data.
+
+    Args:
+        file_path (str): Path to the JUnit XML file.
+
+    Returns:
+        list of dict: A list of dictionaries, each representing a test case.
+    """
+    test_suites = []
+    tree = ET.parse(file_path)
+    root = tree.getroot()
+    
+    for testsuite in root.iter('testsuite'):
+        suite = {
+            'name': testsuite.get('name', ''),
+            'tests': testsuite.get('tests', ''),
+            'failures': testsuite.get('failures', ''),
+            'test_cases': []
+        }
+        
+        for testcase in testsuite.iter('testcase'):
+            case = {
+                'name': testcase.get('name', ''),
+                'classname': testcase.get('classname', ''),
+                'time': testcase.get('time', ''),
+                'status': '✅'
+            }
+
+            failure = testcase.find('failure')
+            error = testcase.find('error')
+            if failure is not None:
+                case['status'] = '❌'
```

Comment:
I'm not sure if it's Github re-interpreting the emoji but we should use the emoji codes rather than the unicode